cmd/controller-manager/app/options/options.go

```diff
@@ -185,8 +185,9 @@ func (o *Options) AddFlags(flags *pflag.FlagSet, allControllers, disabledByDefau
 		"<group> for skip resources with a specific API group(e.g. networking.k8s.io),\n"+
 		"<group>/<version> for skip resources with a specific API version(e.g. networking.k8s.io/v1beta1),\n"+
 		"<group>/<version>/<kind>,<kind> for skip one or more specific resource(e.g. networking.k8s.io/v1beta1/Ingress,IngressClass) where the kinds are case-insensitive.")
-	flags.StringSliceVar(&o.SkippedPropagatingNamespaces, "skipped-propagating-namespaces", []string{},
-		"Comma-separated namespaces that should be skipped from propagating in addition to the default skipped namespaces(karmada-system, karmada-cluster, namespaces prefixed by kube- and karmada-es-).")
+	flags.StringSliceVar(&o.SkippedPropagatingNamespaces, "skipped-propagating-namespaces", []string{"kube-.*"},
+		"Comma-separated namespaces that should be skipped from propagating in addition to the default skipped namespaces 'kube-.*'.\n"+
```

Comment:
```suggestion
	flags.StringSliceVar(&o.SkippedPropagatingNamespaces, "skipped-propagating-namespaces", []string{"kube-.*"},
		"Comma-separated namespaces that should be skipped from propagating.\n"+
			"Note: 'karmada-system', 'karmada-cluster', and 'karmada-es-.*' are Karmada reserved namespaces that will always be skipped.")
```
How about this? And I'd like to invite @yizhang-zen for comments.

Reply:
👍 The words look good to me. Shall we mention the `default` namespace also be skipped? https://github.com/karmada-io/karmada/blame/master/pkg/controllers/namespace/namespace_sync_controller.go#L96-L98

Reply:
> Shall we mention the default namespace also be skipped?

Nice finding! But I don't think so.

We are talking about which namespace should be skipped from propagating, actually, we mean the `resources` in such namespaces should be skipped. The reason why the `namespace sync controller` need to skip `kube-*` namespaces is we don't need to because we assume the namespace should already be in the member cluster.

Reply:
Slightly different, for the `default` namespace, Karmada will not be created in the member cluster because the member cluster already has the `default` namespace. However, resources (deployment, pod, etc.) in `default` will still be propagated to member clusters.

For namespaces specified in the `skipped-propagating-namespaces` parameter, these namespaces themselves will not be propagated to member clusters and resources in these namespaces will also not be propagated to member clusters.